src/tiles/hover/tile-hover-pane.ts

```diff
@@ -15,6 +22,10 @@ export class TileHoverPane extends LitElement {
 
   @property({ type: Boolean }) loggedIn: boolean = false;
 
+  @property({ type: Boolean }) mobileBreakpoint?: boolean;
```

Comment:
q: are `currentWidth` & `mobileBreakpoint` numbers or a boolean?

Reply:
These are should be numbers, sorry for typo (just copy pasted above property and forgot to change :grin: